doc/overview.md

```diff
@@ -1,31 +1,29 @@
 # Config transpiler overview
 
-The Container Linux config transpiler ("ct" for short) is the utility responsible for transforming a Container Linux configuration from the human-friendly, YAML form into an [Ignition][ignition] configuration. While it is possible to write Ignition configs directly, CoreOS recommends that this tool be used instead.
+The Config Transpiler, ct, is the utility responsible for transforming a user-provided Container Linux Configuration into an [Ignition][ignition] configuration. The resulting Ignition config can then be provided to a Container Linux machine when it first boots in order to provision it.
 
-[ignition]: https://github.com/coreos/ignition
-
-## Why does Container Linux have two different configs?
+The Container Linux Config is intended to be human-friendly, and is thus in YAML. The syntax is rather forgiving, and things like references and multi-line strings are supported.
 
-Throughout the course of operating instances of Container Linux, users will notice that there are two different configs used with Container Linux: the Container Linux Config (formatted as YAML) and the Ignition Config (formatted as JSON). This may seem odd at first, but using two different configs provides both ease-of-use and power. Before elaborating on this claim, an overview of each formatted is warranted.
+The resulting Ignition config is very much not intended to be human-friendly. It is an artifact produced by ct that users should simply pass along to their machines. JSON was chosen over a binary format to make the process more transparent and to allow power users to inspect/modify what ct produces, but it would have worked fine if the result from ct had not been human readable at all.
 
-### Overview of the config types
+[ignition]: https://github.com/coreos/ignition
 
-#### Container Linux Config
+## Why a two step process?
 
-The Container Linux Config, as the name implies, is specific to Container Linux. It is designed to be easy to read and easy to write, something at which YAML excels. This is the config format that users are intended to directly manipulate. This config format is not explicitly versioned and instead depends on the particular version of ct that is in use. It is assumed that the user will manually keep their Container Linux Configs and ct in sync with one another. Because both the configs and ct are under the users control, this can be an atomic operation and it is unnecessary to explicitly version the Container Linux Config.
+There are a couple factors motivating the decision to not incorporate support for Container Linux Configs directly into the boot process of Container Linux (as in, the ability to provide a Container Linux Config directly to a booting machine, instead of an Ignition config).
 
```

Comment:
could we make this an itemized list, instead of stating "first and foremost",  "second' etc, and cut down the number of words in these sentences?